pilot/pkg/config/clusterregistry/conversion.go

```diff
@@ -0,0 +1,219 @@
+/*
```

Comment:
Please use the same copyright format as all the other files (each line prefixed by `//` rather than block comments).

Reply:
These files need to be moved to a `kube` dir somewhere since they're all k8s specific. Not sure which fits best (`pilot/pkg/kube/clusterregistry`, or `pilot/pkg/config/kube/clusterregistry`). Any types/code that aren't k8s specific can live in `pilot/pkg/config/clusterregistry`.

Reply:
Ok, will fix the copyright format.

This file is actually not intended to be k8s specific. see line 162 in the same file